copotrzebne-pl/copotrzebne-frontend
# Feat/delete org

Add org deletion for admin role. Closes https://github.com/copotrzebne-pl/copotrzebne-frontend/issues/69
Testing:
- log in as admin using `tester` account credentials
- create new test organisation (name, city, street and building number are mandatory)
- go to the main page (logged in), find your organisation - it should have a thrash bin icon on the top right corner; try to delete it - cancel or confirm
- if canceled - should redirect to org panel
- if confirmed - go to the main page - see organisation not present
- log out - no trash bins available anymore
- profit
